gnomad/utils/annotations.py

```diff
@@ -487,14 +492,20 @@ def annotate_freq(
     )
 
     # Annotate columns with group_membership
+    freq_sample_count = mt.aggregate_cols(
```

Comment:
Sorry can't comment on the L494 in github but code the current comment on L494 be moved to L497 and updated for the freq_sample_count? Or removed?